config/migrations/2024/20241125151000-purge-duplicate-modifieds.sparql

```diff
@@ -0,0 +1,29 @@
+  PREFIX dct: <http://purl.org/dc/terms/>
+  PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+
+  DELETE {
+    GRAPH ?h {
+      ?s dct:modified ?other.
+    }
+  }
+  INSERT {
+    GRAPH ?h {
+      ?s dct:modified ?oldest.
+    }
+  }
+  WHERE {
+    GRAPH ?g {
+      ?s dct:modified ?oldest.
+    }
+    GRAPH ?h {
+      ?s dct:modified ?other.
+    }
+    FILTER NOT EXISTS {
+      ?s dct:modified ?evenOlder.
+      FILTER(?evenOlder > ?oldest)
+    }
+    ?g ext:ownedBy ?someone.
+    ?h ext:ownedBy ?someoneElse.
+
+    FILTER (?oldest > ?other)
```

Comment:
These filters are a bit confusing, they seem to say you select the oldest date, but you actually select the most recent one. I guess it does not really matter?